R/dash.R

```diff
@@ -1162,10 +1228,19 @@ Dash <- R6::R6Class(
 
           # reset the timestamp so we're able to determine when the last cycle end occurred
           private$last_cycle <- as.integer(Sys.time())
+
+          # flush the context to prepare for the next request cycle
+          self$server$set_data("timing_information", list())
```

Comment:
[Unlike Flask's request context](https://flask.palletsprojects.com/en/1.1.x/reqcontext/), entries in the Fiery data store is not generally cleared at the end of the request cycle, so `self$server$set_data` appears to be persistent for the life of the session.

Reply:
Feels to me like the `before-request` handler is enough to initialize this data, and if we want to clear it after the request we should just do that in the `request` handler immediately after using this info to set the `Server-Timing` headers.